integrated_channels/integrated_channel/management/commands/transmit_learner_data.py

```diff
@@ -20,19 +21,40 @@ class Command(IntegratedChannelCommandMixin, BaseCommand):
     Transmit Enterprise learner course completion data for the given EnterpriseCustomer.
     ''')
 
+    def add_arguments(self, parser):
+        """
+        Add required --api_user argument to the parser.
+        """
+        parser.add_argument(
+            '--api_user',
+            dest='api_user',
+            required=True,
+            metavar=_('LMS_API_USERNAME'),
+            help=_('Username of a user authorized to fetch grades from the LMS API.'),
+        )
+        super(Command, self).add_arguments(parser)
+
     def handle(self, *args, **options):
         """
         Transmit the learner data for the EnterpriseCustomer(s) to the active integration channels.
         """
+        # Ensure that we were given an api_user name, and that User exists.
+        api_username = options['api_user']
```

Comment:
Yes.  But if we move to using the [Certificates API](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/certificates/apis/v0/views.py#L70) instead of accessing the GeneratedCertificate model directly, we'll need a username for that query too.